src/components/Footer.tsx

```diff
@@ -1,18 +1,54 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '../store';
+import { StandartItem, PropsType } from '../types/types';
+import { deleteAllItemsCreator, filterCreator } from '../store/actions';
 
-interface Props {
-  amount: number;
-}
+function Footer({ checkAll, setCheckAll }: PropsType): JSX.Element {
```

Comment:
Just a nitpick - `checkAll` sounds like it's a function. If it's just a boolean variable - call it allChecked and the corresponding method setAllChecked.
Also `PropsType` is not a good name for an interface. Follow this convention - component is called CustomComponent -> interface for its props should be called CustomComponentProps.